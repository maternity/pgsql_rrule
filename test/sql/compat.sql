BEGIN;

SELECT get_freq('FREQ=WEEKLY');
SELECT get_count('FREQ=WEEKLY;COUNT=7');
SELECT get_until('FREQ=WEEKLY;UNTIL=19970805');
SELECT get_interval('FREQ=WEEKLY;INTERVAL=4');
SELECT get_occurrences('FREQ=WEEKLY;BYDAY=SA,SU', date '19970805', date '19970905');

SELECT get_occurrences(rrule('FREQ=WEEKLY;BYDAY=SA,SU'), date '19970805', date '19970905');

ROLLBACK;
