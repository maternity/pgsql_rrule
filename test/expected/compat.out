BEGIN;
SELECT get_freq('FREQ=WEEKLY');
 get_freq 
----------
 WEEKLY
(1 row)

SELECT get_count('FREQ=WEEKLY;COUNT=7');
 get_count 
-----------
         7
(1 row)

SELECT get_until('FREQ=WEEKLY;UNTIL=19970805');
 get_until 
-----------
 19970805
(1 row)

SELECT get_interval('FREQ=WEEKLY;INTERVAL=4');
 get_interval 
--------------
            4
(1 row)

SELECT get_occurrences('FREQ=WEEKLY;BYDAY=SA,SU', date '19970805', date '19970905');
                                      get_occurrences                                      
-------------------------------------------------------------------------------------------
 {08-09-1997,08-10-1997,08-16-1997,08-17-1997,08-23-1997,08-24-1997,08-30-1997,08-31-1997}
(1 row)

ROLLBACK;
